metadata-transformer/src/main/java/no/unit/nva/metadata/service/MetadataService.java

```diff
@@ -73,23 +75,54 @@ private ByteArrayInputStream loadExtractedData() {
         }
     }
 
+    private ByteArrayInputStream loadExtractedData() {
+        return new ByteArrayInputStream(translatorService.getOutputStream().toByteArray());
+    }
+
     private Model getMetadata(URI uri) throws ExtractionException, IOException, URISyntaxException {
         translatorService.loadMetadataFromUri(uri);
         try (RepositoryConnection repositoryConnection = db.getConnection()) {
             repositoryConnection.add(loadExtractedData(), EMPTY_BASE_URI, RDFFormat.JSONLD);
-            return QueryResults.asModel(repositoryConnection.getStatements(null, null, null));
+            try (RepositoryResult<Statement> statements = repositoryConnection.getStatements(null, null, null)) {
+                return normalizeStatements(statements);
+            }
         } finally {
             db.shutDown();
         }
     }
 
+    private Model normalizeStatements(RepositoryResult<Statement> statements) {
```

Comment:
More general, but maybe we want to normalize more stuff?

Reply:
That was my thought too so +1